src/spyglass/common/common_task.py

```diff
@@ -168,14 +178,64 @@ def make(self, key):
 
                     key["epoch"] = epoch
                     target_interval = str(epoch).zfill(2)
+                    target_interval_name = None
                     for interval in session_intervals:
                         if (
                             target_interval in interval
                         ):  # TODO this is not true for the beans file
```

Comment:
I'm not sure which file this would be.  There's no entry in the lab database with a `nwb_file_name` or `subject_id` like 'beans'.  I'm guessing this was an early dev file before the `minirec` one (?)